src/components/state-details.js

```diff
@@ -534,22 +516,6 @@ export default function StateDetailsPage ({ location, data }) {
               />
             </div>
 
-            <div className="action-panel">
```

Comment:
Now that there are no more action-panels, can you remove any CSS related to `.action-panel`? Unless this is still used somewhere, but I'm 95% sure it's just on this page.